modules/button/css/lib/button.scss

```diff
@@ -7,104 +7,53 @@ $_wdc-btn-caret-down-icon-hover: svg-load(
   fill=$wdc-btn-icon-hover-color
 );
 
-@mixin wdc-btn-icon-inline($_icon) {
-  content: '';
-  background-image: $_icon;
-  background-position: center;
-  background-size: $wdc-spacing-s;
-  background-repeat: no-repeat;
-}
-
-@mixin wdc-btn-box-shadow($color) {
-  box-shadow: 0 0 0 $wdc-btn-border-size $color;
-}
+$_wdc-btn-caret-down-icon-white: svg-load(
+  '#{$_wdc-btn-icons-path}/wd-icon-caret-down.svg',
+  fill=$wdc-btn-icon-color-white
+);
 
 @mixin wdc-btn() {
   @include wdc-type-body();
   @include wdc-type-variant-btn();
   display: inline-block;
   text-decoration: none;
-  font-family: $wdc-font-family;
-  font-weight: $wdc-btn-font-weight;
-  min-width: $wdc-btn-min-width-l;
+  min-width: 96px;
   max-width: $wdc-btn-max-width-l;
   height: $wdc-btn-height-l;
-  padding: $wdc-btn-padding-l;
-  background-color: $wdc-btn-bg;
-  border: 1px solid $wdc-btn-border;
-  color: $wdc-btn-color;
-  border-radius: $wdc-btn-border-radius;
-  transition: all 120ms, box-shadow 100ms;
+  padding: 0 14px;
```

Comment:
This doesn't account for the nested span with 8px of padding in the react buttons. This should be `24px`. The React version is actually 2px off. Can you fix this in #191 (Large should be 24 + 8, Medium should be 16 + 8 and Small should be 16 https://github.com/Workday/canvas-kit/blob/master/modules/button/react/lib/ButtonStyles.ts#L232)?